apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx

```diff
@@ -245,119 +245,103 @@ export const UpdateSkillFamily = ({
+                      description:
+                        "Label displayed on the create a skill family form description (English) field.",
+                    })}
+                    rules={{
+                      required: intl.formatMessage(errorMessages.required),
+                    }}
+                  />
+                  <TextArea
+                    id="description_fr"
+                    name="description.fr"
+                    label={intl.formatMessage({
+                      defaultMessage: "Description (French)",
+                      id: "Q0gIlv",
+                      description:
+                        "Label displayed on the create a skill family form description (French) field.",
+                    })}
+                    rules={{
+                      required: intl.formatMessage(errorMessages.required),
+                    }}
+                  />
+                </div>
+                <div className="my-6 flex flex-col gap-6">
+                  <Combobox
+                    id="skills"
+                    name="skills"
+                    isMulti
+                    label={intl.formatMessage(adminMessages.skills)}
+                    placeholder={intl.formatMessage({
+                      defaultMessage: "Select one or more skills",
+                      id: "GhszAa",
+                      description:
+                        "Placeholder displayed on the skill family form skills field.",
+                    })}
+                    options={skillOptions}
+                  />
 
-                <FieldDisplay label={intl.formatMessage(adminMessages.key)}>
-                  {skillFamily.key ??
-                    intl.formatMessage(commonMessages.notProvided)}
-                </FieldDisplay>
-              </div>
-              <CardSeparator />
-              <div
-                data-h2-display="base(flex)"
-                data-h2-gap="base(x1)"
-                data-h2-flex-direction="base(column) p-tablet(row)"
-                data-h2-align-items="base(center)"
-                data-h2-text-align="base(center) p-tablet(inherit)"
-              >
-                <Submit text={intl.formatMessage(formMessages.saveChanges)} />
-                <Link
-                  color="warning"
-                  mode="inline"
-                  href={paths.skillFamilyView(skillFamily.id)}
-                >
-                  {intl.formatMessage(commonMessages.cancel)}
-                </Link>
-              </div>
-            </form>
-          </FormProvider>
-        </Card>
+                  <FieldDisplay label={intl.formatMessage(adminMessages.key)}>
+                    {skillFamily.key ??
+                      intl.formatMessage(commonMessages.notProvided)}
+                  </FieldDisplay>
+                </div>
+                <CardSeparator />
+                <div className="flex flex-col items-center gap-6 text-center xs:flex-row xs:text-inherit">
```

Comment:
@esizer Should this be changed as well then?

```suggestion
                <div className="flex flex-col items-center gap-6 text-center xs:flex-row xs:text-left">
```

Reply:
oh yeah. I missed that one :+1: